source/class/cv/plugins/TR064/calllist.js

```diff
@@ -0,0 +1,378 @@
+/* calllist.js 
+ * 
+ * copyright (c) 2018, Christian Mayer and the CometVisu contributers.
+ * 
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+ */
+
+
+/**
+ * The TR-064 plugin and widget creates a interface to routers that are
+ * configured by the TR-064 protocol, like the well known Fritz!Box routers.
+ * 
+ * The protocol is described at: https://avm.de/service/schnittstellen/
+ * 
+ * @author Christian Mayer
+ * @since 0.11.0
+ */
+/*
+
+https://fritz.box:49443/tr64desc.xml
+
+http://wiregate/CometVisuGit/source/resource/plugins/TR064/soap.php?device=fritzbox
+&location=upnp/control/x_tam&uri=urn:dslforum-org:service:X_AVM-DE_TAM:1&fn=GetMessageList&p[]=NewIndex&v[]=0
+&location=upnp/control/x_contact&uri=urn:dslforum-org:service:X_AVM-DE_OnTel:1&fn=GetInfoByIndex&p[]=NewIndex&v[]=1
+&location=upnp/control/x_contact&uri=urn:dslforum-org:service:X_AVM-DE_OnTel:1&fn=GetCallList
+&location=upnp/control/wlanconfig1&uri=urn:dslforum-org:service:WLANConfiguration:1&fn=GetInfo
+
+&location=&uri=&fn=&p[]=&v[]=
+
+*/
+qx.Class.define('cv.plugins.TR064.calllist', {
+  extend: cv.ui.structure.AbstractWidget,
```

Comment:
Just one thing about "package" and Class naming conventions:
- The currently used convention is: lower-case package names and CamelCase class names. This has been used throughout the whole CometVisu and we should stick to it!

So please rename this one to `cv.plugins.tr064.CallList`

Reply:
Everything is renamed now

Reply:
The classname itself is still lowercase and no camelcase: should be `cv.plugins.tr064.CallList`

Reply:
and don't forget to change the filename too then (CallList.js)

Reply:
Done